src/rust/engine/nailgun/src/client.rs

```diff
@@ -155,7 +152,7 @@ pub async fn client_execute(
     stdin_read
   })
   .await
-  .map_err(|err| NailgunClientError::PreConnect(format!("Failed to start remote task: {}", err)))?;
+  .map_err(|err| NailgunClientError::PreConnect(format!("Failed to start: {}", err)))?;
```

Comment:
Not sure what this wording should be. Really, it's a failure to start the Pants command, meaning sending the sys.argv to Nailgun like `./pants lint ::`.